addons/payment/views/payment_templates.xml

```diff
@@ -63,6 +63,12 @@
                                        t-att-checked="acquirers_count==1 and pms_count==0 or acquirers[0] == acq"/>
                             </t>
                             <span class="payment_option_name" t-esc="acq.name"/>
+                            <t t-if="acq_extra_fees and acq_extra_fees.get(acq)">
+                                <span class="badge badge-primary"> + <t t-esc="acq_extra_fees[acq]" t-options='{"widget": "monetary", "display_currency": acq_extra_fees["currency_id"]}'/> Fee </span>
```

Comment:
`badge-secondary` to fit other badge price style (eg variant extra on product page)

Reply:
done.!